frontend/src/pages/Home/MailingListsPage.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from "react";
+import { Table } from 'reactstrap';
+import BasePage from '../BasePage';
+
+class MailingListsPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {loading: false}
+    }
+
+    render() {
```

Comment:
This looks good! The Simmons Mailman lists don't work though so would you be able to use [these lists](https://docs.google.com/spreadsheets/d/1zRI-rJSJjb4GHPbOAxo_XQXeqHHKudAp_hSxDdx_rSk/edit?usp=sharing) instead? Sorry about that! If you could use the columns on the spreadsheet too that would be great :)

Reply:
I'm sorry about that! It should be updated now (I can work on making it pull info directly from the sheets, but for now it's just a copy-paste table)

Reply:
looks good!